src/views/settings/CategorizationSettings.vue

```diff
@@ -61,6 +66,33 @@ export default {
     restoreDefaultClasses: async function () {
       await this.$store.commit('categories/restoreDefaultClasses');
     },
+    exportClasses: function () {
+      console.log('Exporting categories...');
+
+      const text = JSON.stringify(this.classes_hierarchy);
```

Comment:
I don't think `classes_hierarchy` is the right thing to export. Just export the list of categories, as they are laid out in `localStorage`.